pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -877,6 +882,15 @@ func (oe *operationExecutor) generateMountVolumeFunc(
 			}
 		}
 
+		canMountErr := volumeMounter.CanMount()
```

Comment:
nit: Let's not even call `CanMount()` if `checkNodeCapabilitiesBeforeMount == false`. It doesn't make much of a difference, but in the case that there is a bug in a `CanMount()` implementation that causes a panic, for example, `checkNodeCapabilitiesBeforeMount` can then be used to completely avoid the check (vs the way it is `CanMount()` would always be called).